docs/guides/place-my-order.md

```diff
@@ -840,36 +842,38 @@ This is a longer template so lets walk through it:
 
 can-connect makes it very easy to implement real-time functionality. It is capable of listening to notifications from the server when server data has been created, updated, or removed. This is usually accomplished via [websockets](https://en.wikipedia.org/wiki/WebSocket), which allow sending push notifications to a client.
 
-### Adding real-time events to a model
+### Update the template
```

Comment:
We're probably going to have some merge conflicts between this and https://github.com/donejs/donejs/pull/817.